doc/source/user_guide/advanced.rst

```diff
@@ -851,11 +851,11 @@ values **not** in the categories, similarly to how you can reindex **any** panda
 Int64Index and RangeIndex
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. note::
-
+.. deprecated:: 1.4.0
     In pandas 2.0, :class:`NumericIndex` will become the default index type for numeric types
     instead of ``Int64Index``, ``Float64Index`` and ``UInt64Index`` and those index types
-    will be removed. See :ref:`here <advanced.numericindex>` for more.
+    are therefore deprecated and will be removed in Pandas 2.0.
```

Comment:
instead of saying Pandas 2.0, just a future version (we don't explicitly list the removal versions anywhere else i believe)

Reply:
same comment applies to all docs below

Reply:
Ok, done.

Reply:
did you push the update?